client/trino-cli/src/main/java/io/trino/cli/QueryRunner.java

```diff
@@ -166,6 +175,32 @@ private static void setupBasicAuth(
         }
     }
 
+    private static void setupExternalAuth(
+            OkHttpClient.Builder builder,
+            ClientSession session,
+            boolean enabled,
+            Consumer<OkHttpClient.Builder> sslSetup)
+    {
+        if (enabled) {
```

Comment:
nit: I think it's better to limit the indentation by using:
```
if (!enabled) return;
```
whenever possible.

Reply:
true.